test/typing/component-object-export.spec.ts

```diff
@@ -9,22 +9,13 @@ export interface RandomComponentProps {
   title: string
 }
 
-export interface RandomComponent
```

Comment:
This represents as mentioned before a breaking change not compatible with the current compiler output